src/status_im2/common/standard_authentication/password_input/view.cljs

```diff
@@ -49,6 +49,7 @@
          {:container-style style/auth-button
           :on-press        on-press-biometrics
           :icon-only?      true
+          :background      :blur
```

Comment:
this should probably be passed as a prop and defaulted to false. wdyt? I think in some flows this could be incorrect 🤔

Reply:
The text input has `:blur?` enabled as well. Do we have cases where the input is used without the blur?

Reply:
here is the designs for syncing auth - https://www.figma.com/file/V6nlpAWIf2e1XU8RJ9yQPe/Syncing-for-Mobile?type=design&node-id=1112-146720&mode=design&t=ioE4htigtApMxrtn-4 
which is in blur and in this it has it.

for wallet:
https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=2340-519746&mode=design&t=JoUGHMyoLfjV00Ay-4

in wallet it is not using blur
<img width="411" alt="Screenshot 2023-12-11 at 14 35 08" src="https://github.com/status-im/status-mobile/assets/22799766/b0b045b3-957c-4d68-b08f-7990ebb6dc4e">